### ISSUE TITLE
Adding attachment removes read access from Moodle role (Paella Player)

### ISSUE THREAD

**Issue by t-schroeder** [opencast/opencast#1385]
(This has only been tested with the Paella Player.)

**Describe the bug**
I have uploaded a video through Moodle [0]. The event is published and has an ACL with entries like

```
{
  "123_Learner": "read",
  "123_Instructor": "read,write"
}
```

When I add an attachment (captions/vtt) to the event through the Opencast admin website, the video can no longer be viewed through Moodle [1]. It doesn't show up anymore under /engage/ui/index.html, either, when authenticated through Moodle/LTI with the role "123_Learner". In the admin website the ACL of the video is still show as unchanged. All access rights are correct according to it.

**To Reproduce**
See above.

**Environment Information**
 - OS: CentOS 7
 - Browser: Chrome
 - Opencast version: 8.0

**Workarounds**
When I then run the default "republish-metadata" workflow, I can access the video through Moodle again. So it seems like the ACL just needs to be republished.

If I adjust the configuration of the default "publish-uploaded-assets" workflow by appending ",security/*" to the value of "download-source-flavors" I don't need to run the republish-metadata workflow after adding the asset.

[0] https://github.com/unirz-tu-ilmenau/moodle-block_opencast
[1] https://github.com/unirz-tu-ilmenau/moodle-filter_opencast

**Comment by ebbertd** [comment]
This should be fixed in Opencast 8.7. Can you confirm that? See PR #1812

**Comment by t-schroeder** [comment]
I can't test it right now. But since that PR includes the same change I mentioned as my second workaround (which I already confirmed fixes the problem back when I created this issue), I would consider this fixed.
